### DISCUSSION TITLE
💡 Wondering how you can stitch your web & mobile user stories together? Catch us discuss the new feature LIVE!

### DISCUSSION THREAD

**Discussion by lvnya-c** [testsigmahq/testsigma#56] [Announcements]
**[📍](https://emojipedia.org/round-pushpin/) The highlights:**

- Now communicate between the browser on your laptop and your mobile application
- Send data for comparison between the two
- Continue testing on the web from where you left off on your mobile device
- Easily switch the context between the web/mobile test case flow
- No need to manage your Selenium and Appium drivers separately

We are building integrated E2E user journey tests for multi-device applications and we'd [❤️](https://emojipedia.org/red-heart/) your feedback!

### Register for the webinar [**here**](https://testsigma.com/webinars/automate-web-mobile-user-journey-end-to-end?utm_source=github&utm_medium=discussions)!

Post your questions below and like/upvote questions so the team knows what’s most important to you!

**Comment by manikkalai-kannan** [comment]
Does this feature support the different browsers from mobile? can we switch context between browser from laptop and browser from mobile?

**Reply by lvnya-c** [reply]
Sure, you can! Do join us in our upcoming [webinar](https://testsigma.com/webinars/automate-web-mobile-user-journey-end-to-end) where we demonstrate how this is done.
